File: src/main/java/Level1/Ex8.java
package Level1;

import java.util.Arrays;
import java.util.Collections;

//문자열 내림차순으로 배치하기
public class Ex8 {
    public static String solution(String s) {
        String answer = "";
        String[] array = s.split("");
        Arrays.sort(array, Collections.reverseOrder());
        answer = String.join("", array);
        return answer;
    }

    public static void main(String[] args) {
        String s = "abegiurbABCABC";
        System.out.println(solution(s));
    }
}

File: src/main/Javascript/Level1/Ex8.js
function solution(s) {
    var answer = '';
    let str = s.split("");
    str.sort(function (a, b) {
        if (a > b) {
            return -1;
        } else if (b > a) {
            return 1;
        } else return 0;
    });
    for (let i = 0; i < str.length; i++) {
        answer += str[i];
    }
    return answer;
}

console.log(solution("Zbcdefg"));